src/main/java/com/fvv/bookstore/util/Constants.java

```diff
@@ -54,4 +54,9 @@
 	 * Constant that represents a date with time mask.
 	 */
 	public static final String DATE_TIME_MASK = "dd-MM-yyyy HH:mm:ss";
+	
+	/**
+	 * Constant that represents the file connection factory properties name.
```

Comment:
"constant holding the name of the file containing the database connection settings"